codegen/templates/chart/operator-deployment.yamltmpl

```diff
@@ -97,10 +101,16 @@ spec:
         - name: {{ $[[ $operatorVar ]]Image.pullSecret }}
       {{- end}}
 
+{{- end }} {{/* define "[[ $operator.Name ]].deploymentSpec" */}}
 
----
+{{/* Render [[ $operator.Name ]] deployment template with overrides from values*/}}
+{{- $[[ $operator.Name ]]DeploymentOverrides := dict }}
```

Comment:
If the oprerator.Name is something with a hypen like 'cert-agent', this will resolve to:
```
{{- $cert-agentDeploymentOverrides := dict }}
```
and this error:

Error: parse error at (cert-agent/templates/deployment.yaml:86): bad character U+002D '-'

Reply:
Replacing with $[[ $operatorVar ]] will fix it temporarily, but I'm not sure if that $[[ $operatorVar ]] has a guarantee of being in camelcase with no hyphens.